### DISCUSSION TITLE
A separate channel for post breakout chats?

### DISCUSSION THREAD

**Discussion by shreemaan-abhishek** [EddieHubCommunity/EddieBotBreakout#26] [Ideas]
It'd be great to have a separate channel for chatting with folks post breakout sessions. I'd love to have your views on this.

**Comment by naomi-lgbt** [comment]
I think this is something that a server would create for themselves, rather than being managed by the bot.

**Comment by shreemaan-abhishek** [comment]
Yes, that's what I'm trying to say.
